README.md

```diff
@@ -11,7 +11,7 @@ Microsoft.Data.SqlClient is a .NET data provider for [Microsoft SQL Server]([url
 The Microsoft.Data.SqlClient package supports the following environments:
 
 - .NET Framework 4.6.2+
-- .NET 8.0
+- .NET 8.0+

```

Comment:
Should we call out .NET 9.0 here explicitly?  When I see "8.0+" I think "Any future 8.x version".

Reply:
You're not calling out .NET Framework 4.7 and 4.8, right?
I think 8.0+ is fair

Reply:
.NET 8.0+ seems perfectly clear to me (any future .NETs)

Reply:
Sure we can call out 9.0 explicitly here. It's a bit confusing since right above we say 4.6.2+ which, to me, says anything greater than 4.6.2 (4.8, etc).

The other thing is even if we explicitly build for .NET 4.6.2, eg, it'll totally work with later versions. So, to me, saying "+" to everything seems a bit redundant 🤷‍♂️